CMakePresets.json

```diff
@@ -0,0 +1,23 @@
+{
+  "version": 3,
+  "configurePresets": [
+    {
+      // Don't set "generator" here because developers should still be able to
+      // select their preferred generators. As a result we're affected by CMake
+      // bug https://gitlab.kitware.com/cmake/cmake/-/issues/23341, which means
+      // that dev-mode is not selectable in cmake-gui. But in our case, that's
+      // probably rather a feature than a bug.
```

Comment:
GitHub highlights these lines because JSON doesn't have comments. But some parsers are very forgiving, and CMake's parser seems to be of that kind.